<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cходство между файлами</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить документов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExecute">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>547</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск сходства</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelExecute">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Панель выполнения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сравнение документов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Инструментальное программное обеспечение разработки и проектирования информационных систем</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ИКМО 01-20     Ранди Реина Эрнандес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>650</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Москва 2021</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteFile">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>447</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить документ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>------------------------------------------------------------------------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>601</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddFileOriginal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить документ</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
